<?xml version="1.0" encoding="UTF-8"?>
<div class="supplementary-material" content-type="local-data" id="PMC7941931/supplementary_9" xmlns="http://www.w3.org/1999/xhtml">
 <div class="media" xlink:href="12870_2021_2914_MOESM1_ESM.zip" xmlns:xlink="http://www.w3.org/1999/xlink">
  <caption> Additional file 1: Table S1. Characteristic features of all the identified DCL, AGO, HYL1, SE, HST, and RDRs in tartary buckwheat genome. Table S2. Length distribution of sRNA sequences identified in developing tartary buckwheat seeds. Table S3. Information of identified conserved miRNAs. Table S4. Information of identified novel miRNAs. Table S5. Predicted target genes of miRNAs in tartary buckwheat. Table S6. Annotation of miRNAs target genes in tartary buckwheat. Table S7. Information of TFs targeting by miRNAs. Table S8. miRNAs have target genes are the orthologs of the known seed or organ size. Table S9. miRNAs have target genes are the orthologs of the known structural or regulatory genes of flavonoid biosynthesis. Table S10. KEGG pathways of the target genes of DEMs. Table S11. miRNA-mRNA interaction pairs show expression negative correlation during tartary buckwheat seed development. Table S12. Primers of sequences for qRT-PCR analysis. Table S13. Primers of sequences for RLM-5â€²RACE analysis. </caption>
 </div>
</div>
