<?xml version="1.0" encoding="UTF-8"?>
<div class="fig" id="PMC7923665/figure_1" xmlns="http://www.w3.org/1999/xhtml">
 <caption> Transcriptome analysis of cerebral AVM nidus and controls tissues. a Pie diagram showing gene distribution of differentially expressed genes in arterio venous malformation (AVM) nidus samples compared to control sample. b The scatter plot representation of differentially expressed genes in cerebral AVM nidus, which allows to identify the expression levels of genes in two distinct conditions. In the scatter plot, each dot represents a gene, and thus, genes that fall above the diagonal are overexpressed and genes that fall below the diagonal are underexpressed in AVM. c Top 100 differentially expressed genes in the cerebral AVM tissues compared to control brain tissues are represented as heat map. The asterisk marked genes represent the genes involved in blood vessel development. The color represents the logarithmic intensity of the expressed genes. Relatively high expression values are shown in red color. d The bar plot represents the Log2 fold change in the expression of commonly differentially expressed genes in the cerebral AVM samples compared to control samples. A total of 38 genes were differentially expressed commonly in all the three independent AVM samples analyzed compared to control samples, among which 35 genes were upregulated and three genes viz, NR5A1, SSTR1, and DUX4L4, were downregulated. e Validation of differentially expressed genes common in three independent samples of cerebral AVM nidus. The following genes were validated by qRT-PCR. COL3A1, CYR61, OSM, MAFB, COL1A1, CTSS, TLR2, and CXCR4, and their expression were normalized to control samples, and internal normalization was carried out using GAPDH. Many tested genes were significantly up regulated in AVM tissue. The LIF and MGP gene expression were not significantly increased in AVM compared to control tissue. The qRT-PCR was carried out with 10 AVM nidus and 10 control samples, and the GAPDH was used as endogenous control for normalization. Individual paired t test was performed to calculate the P values (* P &lt; 0.05 and ** P &lt; 0.01) </caption>
 <div class="graphic"/>
 <p class="label">Fig. 1</p>
</div>
