<?xml version="1.0" encoding="UTF-8"?>
<div class="fig" id="PMC7913442/figure_4" xmlns="http://www.w3.org/1999/xhtml">
 <caption> Effect of FOXO1 overexpression on FOXL2 C134W – modified genes. a Venn diagram showing the number of differential genes (DESeq, q &lt; 0.05) identified in FOXL2 C134W vs FOXL2 WT and FOXL2 C134W + FOXO1 vs FOXL2 WT. b Scatter plot of the effect sizes (log2-fold change) of DEGs between FOXL2 C134W and FOXL2 WT, with (y-axis) and without (x-axis) the addition of FOXO1. Genes that were differentially expressed also when adding FOXO1 3A are shown in black (n = 130), while genes that were differentially expressed only without FOXO1 3A addition are shown in purple (n = 587). The slope (ß) of the linear relationship between the two effect sizes is shown on top of the plot. c Venn diagram showing the number of differential genes (DESeq, q &lt; 0.05) identified in FOXL2 C134W vs FOXL2 WT and FOXL2 C134W + FOXO1 vs FOXL2 C134W. d Heatmap with gene clustering of the 39 DEGs in both FOXL2 C134W vs FOXL2 WT and FOXL2 C134W + FOXO1 vs FOXL2 C134W comparisons. For each gene, expression values were z-score normalized across samples. Genes are annotated for presence or absence of a FOXL2 C134W ChIP-seq binding site in TGFβ-treated HGrC1 (ChIP1) [ 37] and a FOXL2 C134W ChIP-seq binding site in a SVOG3e cell line (ChIP 2) [ 28]. e Metascape GO analysis of the 39 DEGs in both FOXL2 C134W vs FOXL2 WT and FOXL2 C134W + FOXO1 vs FOXL2 C134W comparisons. The red line indicates p-value cutoff for p &lt; 0.05. GO terms associated with expression of FOXL2 and/or SMAD3 were removed from the results (see Methods). f qPCR validation of the RNAseq expression data in FOXO1 modulated genes (BAZ2A, SMARCA4, COL4A2, and HSPG2). Graphs show fold change means ± s.e.m relative expression to FOXL WT following normalization to the housekeeping gene. Data were analyzed using the two-tailed unpaired t-test (*p &lt; 0.05, **p &lt; 0.01, ***p &lt; 0.001) </caption>
 <div class="graphic"/>
 <p class="label">Fig. 4</p>
</div>
