<?xml version="1.0" encoding="UTF-8"?>
<div class="fig" id="PMC7913442/figure_3" xmlns="http://www.w3.org/1999/xhtml">
 <caption> Identification of direct FOXL2 targets among DEGs between FOXL2 C134W and FOXL2 WT. a Enrichment of FOXL2 binding in DEGs between FOXL2 C134W and FOXL2 WT with respect to non-DEGs using FOXL2 C134W ChIP-seq binding site in TGFβ-treated HGrC1 (ChIP1) [ 37] and another b using FOXL2 C134W ChIP-seq binding site in SVOG3e cell line (ChIP 2) [ 28]. p-values and odds ratios (OR) are from two-tailed Fisher’s exact test. c Heatmap of the 717 DEGs between FOXL2 C134W and FOXL2 WT grouped into 6 categories based on direction of expression in FOXL2 C134W, FOXL2 WT and empty vector, as depicted in the diagrams. For each category, the top 30 DEGs (or less for smaller groups) are listed. d Enrichment of differential FOXL2 C134W vs FOXL2 WT binding (see Methods) at promoters of genes in each of the 6 DEG categories, color-coded as in ( c). p-values and odds ratios (OR) are from two-tailed Fisher’s exact test. e Genome-browser screenshots of FOXL2 WT and FOXL2 C134W ChIP-Seq in TGFβ-treated HGrC1 [ 37], showing differential binding at the TGFB2 (top), SMARCA4 (middle) and NFKBIA (bottom) promoter regions. f Venn diagram (upper panel) showing the number of differential genes (q &lt; 0.1) identified in FOXL2 C134W vs FOXL2 WT (20 DEGs) from the present study and from adult granulosa cell tumor patient samples vs control samples (111 DEGs) previously published [ 25]. The lower panel shows heatmap of the common identified 11 genes. For each gene, expression values were z-score normalized across samples. Genes are annotated for presence or absence of a FOXL2 C134W ChIP-seq binding site in TGFβ-treated HGrC1 (ChIP1) [ 37] and a FOXL2 C134W ChIP-seq binding site in a SVOG3e cell line (ChIP 2) [ 28] </caption>
 <div class="graphic"/>
 <p class="label">Fig. 3</p>
</div>
