<?xml version="1.0" encoding="UTF-8"?>
<div class="fig" id="PMC7822815/figure_7" xmlns="http://www.w3.org/1999/xhtml">
 <caption> RUNX1/RUNX1T1 affects differential splicing by controlling alternative transcription start sites (TSSs). a Vulcano plot of the differential expression from various TSSs in the genome of Kasumi-1 cells following RUNX1/RUNX1T1 knockdown. Differentially expressed TSSs (at least 2-fold change in expression, p &lt; 0.005, q &lt; 0.1) are shown by vermilion squares. b Differential usage of alternative TSSs in the genome of Kasumi-1 cells after RUNX1/RUNX1T1 knockdown. Differentially used TSSs (at least 2-fold change in expression, p &lt; 0.0007, q &lt; 0.1) are shown by vermilion squares. c Enrichment of DNase I hypersensitivity site-Seq reads in the genomic regions surrounding the non-differential TSSs, differentially expressed TSSs or differentially used TSSs. In this analysis, only TSSs genomic regions that overlap the RUNX1/RUNX1T1 binding peaks were used. In each boxplot, horizontal line represents the median of expression distribution, box shows the interquartile range, and whiskers are the minimum and maximum. P-values were calculated with two-sided Mann–Whitney U test. d IGV screen shot showing the RUNX1/RUNX1T1 binding and RNA reads from nascent and total RNA-Seq at the 5’-terminal region of PARL locus for siMM-treated and siRR-treated Kasumi-1 cells. Exons and exon-exon junctions are designated by the letters E and J, respectively, and numbered. Differentially used junction J3 is highlighted in purple. Bottom scheme depicting Cufflinks assembled representative full-length transcripts of PARL gene expressed in Kasumi-1 cells. For each transcript, Cuffdiff based log 2 (FC) siRR/siMM and q-values, as well as the size of in silico predicted proteins are indicated in parentheses. In addition, the positions of the Cuffdiff determined transcription start sites are also shown. At the bottom of panel, the scheme showing target location of dCas9-KRAB which was targeted to the second RUNX1 consensus site. e Schematic representation of primer binding positions for qPCR-based validation of the differential splicing in PARL gene. The structure of expected amplicons is shown at the bottom of the figure. f qPCR-based validation of the differential splicing of PARL transcripts under the two siRNA treatment conditions in the transcriptome of two t(8;21)-positive cell lines. Bars represent mean ± SD of three independent experiments. g Induction of PARL-03 transcription after RUNX1/RUNX1T1 knockdown with or without induction of dCas9-KRAB. Bars represent mean ± SD of three independent experiments. P-value was calculated with two-sided Student’s t test. </caption>
 <div class="graphic"/>
 <p class="label">Fig. 7</p>
</div>
