<?xml version="1.0" encoding="UTF-8"?>
<div class="fig" id="PMC7822815/figure_9" xmlns="http://www.w3.org/1999/xhtml">
 <caption> RUNX1/RUNX1T1 indirectly affects differential splicing in Kasumi-1 cells. a Bar graph showing the impact of RUNX1/RUNX1T1 knockdown on the expression of mRNA surveillance genes (SurG), genes encoding splicing factors (SplG) and transcription factors genes (TFG) associated with splicing. Bars represent mean ± SD of three independent qPCR experiments. * p &lt; 0.05, ** p &lt; 0.01, *** p &lt; 0.001 with one-tailed Student’s t test. b Graph showing correlation between RNA-seq and q-PCR validation of differential RNA processing gene expression. c IGV screen shot depicting RUNX1/RUNX1T1 binding and RNA-Seq reads for the USB1 locus. d Cytoscape scheme showing RUNX1/RUNX1T1-centered gene regulatory network of genes encoding splicing regulators and mRNA surveillance factors. Terms “upregulated genes” and “downregulated genes” refer to a change in the expression of the genes after RUNX1/ RUNX1T1 knockdown. Positively and negatively co-expressed genes linked by green and red edges, respectively. Examples of the genes from each group are shown in the respective dashed boxes. e Effect of the strength of the motifs to splicing factors on the probability of non-diffEEJs and diffEEJs in leukemia cells. These partial dependence plots are based on the results from 1,000 independent runs of the random forest meta-classifier and 1000 classification trees per random forest per run. </caption>
 <div class="graphic"/>
 <p class="label">Fig. 9</p>
</div>
